validator/src/main.rs

```diff
@@ -1361,6 +1362,15 @@ pub fn main() {
                 .takes_value(false)
                 .help("Use the just-in-time compiler instead of the interpreter for BPF."),
         )
+        .arg(
+            Arg::with_name("indexes")
+                .long("index")
```

Comment:
```suggestion
            Arg::with_name("account_indexes")
                .long("account-index")
```
"index" is a little too generic

Reply:
Done in https://github.com/solana-labs/solana/pull/14212/commits/d204a08432c3936cd1450cf16e1a67236e758461